SentinelOne/action_init_scan.json

```
@@ -0,0 +1,55 @@
+{
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+      "ids": {
```

My bad. The parameter for agent ids is `uuid`
```suggestion
      "uuids": {
```